migrating_from_ocp_3_to_4/index.adoc

```diff
@@ -0,0 +1,84 @@
+[id="migration-from-version-3-to-4-overview"]
+= Migration from version 3 to 4 overview
+
+{product-title} 4 clusters are different from {product-title} 3 clusters. {product-title} 4 clusters contain new technologies and functionality that result in a cluster that is self-managing, flexible, and automated To know more about migrating from {product-title} 3 to 4 see, xref:../migrating_from_ocp_3_to_4/about-migrating-from-3-to-4.adoc#about-migrating-from-3-to-4[migrating from version 3 to 4].
+
+[discrete]
+[id="mtc-3-to-4-overview-differences-mtc"]
+=== Differences between {product-title} 3 and 4
+Differences between {product-title} 3 and 4: To successfully transition to {product-title} 4, review the following information:
+
+** Architecture
+** Installation and upgrade
+** Storage, network, logging, security, and monitoring considerations
+
+[discrete]
+[id="mtc-3-to-4-overview-planning-network-considerations-mtc"]
+=== Planning network considerations
+Planning network considerations: Before migrating from {product-title} 3 to 4 plan your network considerations, review the following information:
+
+** DNS considerations
+*** Isolating the DNS domain of the target cluster from the clients.
+*** Setting up the target cluster to accept the source DNS domain.
+
+You can migrate stateful application workloads from {product-title} 3 to 4 at the granularity of a namespace. To know more about MTC see the xref:../migration_toolkit_for_containers/about-mtc.adoc#about-mtc[Understanding MTC].
+
+[NOTE]
+====
+If you are migrating from {product-title} 3, see xref:../migrating_from_ocp_3_to_4/about-migrating-from-3-to-4.adoc#about-migrating-from-3-to-4[About migrating from {product-title} 3 to 4] and xref:../migrating_from_ocp_3_to_4/installing-3-4.adoc#migration-installing-legacy-operator_installing-3-4[Installing the legacy {mtc-full} Operator on {product-title} 3].
+====
+
+[discrete]
+[id="mtc-overview-install-mtc"]
+== Installating MTC
+You must install the Migration Toolkit for Containers that is compatible with your {product-title} version by using OLM. You manually install the legacy Migration Toolkit for Containers Operator on {product-title} 4.5 and earlier versions. You must configure an object storage to use as a replication repository. You can check the following items:
+
+* xref:../migration_toolkit_for_containers/installing-mtc.adoc#installing-mtc[Install migration Toolkit for Containers Operator on {product-title}]
+* xref:../migration_toolkit_for_containers/installing-mtc.adoc#configuring-replication-repository_installing-mtc[Configure replication repository]
+
+[discrete]
+[id="mtc-overview-upgrade-mtc"]
+== Upgrading MTC
+You can xref:../migration_toolkit_for_containers/upgrading-mtc.adoc#upgrading-mtc[upgrade the MTC] by using Operator Lifecycle Manager.
```

Comment:
> You can xref:../migration_toolkit_for_containers/upgrading-mtc.adoc#upgrading-mtc[upgrade the MTC] by using Operator Lifecycle Manager.

The correct info is [here](https://docs.openshift.com/container-platform/4.9/migrating_from_ocp_3_to_4/upgrading-3-4.html):

You upgrade the Migration Toolkit for Containers (MTC) on OpenShift Container Platform 4.9 by using Operator Lifecycle Manager.

You upgrade MTC on OpenShift Container Platform 3 by reinstalling the legacy Migration Toolkit for Containers Operator.

Reply:
Addressed